src/Toolkit/Toolkit.Maui/Internal/HtmlToView.Maui.cs

```diff
@@ -163,6 +163,40 @@ private static View CreateBlock(MarkupNode node, EventHandler<Uri>? urlClickHand
                     imageElement.Source = imageSource;
                 return imageElement;
 
+            case MarkupType.Audio:
+            case MarkupType.Video:
```

Comment:
What happens with this code on Android? Shouldn't that still show the placeholder instead?